docs/source/libraries/keras.rst

```diff
@@ -0,0 +1,40 @@
+.. _library-keras:
+
+Keras
+=====
+
+Keras_ is "a high-level neural networks API, written in Python and capable of running on top of TensorFlow, CNTK, or Theano". 
+
+Currently ELI5 supports :func:`eli5.explain_prediction` for Keras image classifiers.
+
+.. _Keras: https://keras.io/
+
+:func:`eli5.explain_prediction` explains image classifications through `Grad-CAM <https://arxiv.org/pdf/1610.02391.pdf>`_. The :class:`eli5.base.Explanation` object returned has a ``.image`` attribute that represents the  image that is inputted into the model, and a ``.heatmap`` attribute that is a grayscale "localization map" roughly indicating regions of importance in the image for the predicted class.
```

Comment:
Usually we prefer to break long lines, unless it's necessary for formatting. Also probably a small typo her:
```suggestion
:func:`eli5.explain_prediction` explains image classifications through `Grad-CAM <https://arxiv.org/pdf/1610.02391.pdf>`_. The :class:`eli5.base.Explanation` object returned has an ``.image`` attribute that represents the  image that is inputted into the model, and a ``.heatmap`` attribute that is a grayscale "localization map" roughly indicating regions of importance in the image for the predicted class.
```